feature/default_sets.go

```diff
@@ -14,7 +14,7 @@ var defaultSetDesc = setDesc{
 		SetInit:    {}, // I
 		SetNodeAnn: {}, // N
 	},
-	lnwire.GossipQueriesOptional: {
+	lnwire.GossipQueriesRequired: {
```

Comment:
Wouldn't that consequentially mean that we should disconnect a peer if they don't support it?
See https://github.com/lightningnetwork/lnd/blob/4fa483f1bcaa811e5b62494eed2cea05ba0a5ee5/peer/brontide.go#L754.

Reply:
Yes. That's the goal. I'll remove the check you cite.